src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -1172,9 +1172,11 @@ private void buildTableColumnHeaderForRecipientQuestionGiverView(List<ElementTag
             responseRows.add(responseRow);
         }
         
-        responseRows.addAll(getRemainingMissingResponseRows(question, possibleGiversWithoutResponses, 
-                                                            possibleReceiversWithoutResponsesForGiver, 
-                                                            prevGiver, viewType));
+        if (!responses.isEmpty()) {
+            responseRows.addAll(getRemainingMissingResponseRows(question, possibleGiversWithoutResponses, 
+                                                                possibleReceiversWithoutResponsesForGiver, 
+                                                                prevGiver, viewType));
+        }
```

Comment:
Yup, but the error in this case is that I still constructed the pending responses even though there are not shown.

Currently, the behaviour is to show the pending responses for the question once there is at least a single response for the question. I was planning to change the behaviour for showing the pending responses in the issue for showing the instructors an option, should I do that here instead?